micromasters_import/management/commands/import_micromasters_data.py

```diff
@@ -24,18 +24,33 @@ class Sql:
         file_name: str
         raw_sql: str
 
-    help = "Imports Micromasters data"
+    help = "Imports Micromasters data. Specify --num to run a paticular file. e.g. --num 002"
+
+    def add_arguments(self, parser) -> None:
+        parser.add_argument(
+            "--num",
+            type=str,
+            help="specify a file num to run. eg. 002 refers to 002_import_courserun.sql",
+        )
 
     def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        file_num = kwargs["num"]
         sqls = []
 
         self.stdout.write(f"Gathering SQL queries:")
 
-        for root, _, files in os.walk(SQL_FILES_DIR):
-            for file_name in sorted(files):
-                with open(os.path.join(root, file_name), "r") as sql_file:
-                    self.stdout.write(file_name)
-                    sqls.append(self.Sql(file_name, sql_file.read()))
+        if file_num:
+            for file in os.listdir(SQL_FILES_DIR):
+                if file_num in file:
```

Comment:
This is probably open to user error issues, for example if I passed `--num 2`, there's no validation of the length of the input being a zero padded value, so this means any script with '2' in the filename `e.g. 020, 152, etc` would get run. I think you want to either validate the input length or zero-pad it in the command to support either style. Then I think you probably want `file_name.startswith` instead of an `in` check.

Also I think you could simplify this into a single file walk but moving the `if file_num` into the `os.walk` below and making it a:

```python
if file_num and not file_name.startswith(file_num):
    continue
```

Reply:
I didn't think too much of validation since it will be used once. But I see your point here. I will make the change